src/client/driver/status.js

```diff
@@ -4,6 +4,7 @@ export default class DriverStatus extends Assignable {
     constructor (obj) {
         super(obj);
 
+        this.id              = Date.now();
```

Comment:
`Date.now()` might have low precision. I suggest using [Performance.now](https://developer.mozilla.org/ru/docs/Web/API/Performance/now) if it's available and fallback to `Date.now() + Math.random()` if not.